pkg/nrop/nrop_scheduler_test.go

```diff
@@ -0,0 +1,350 @@
+package nrop
+
+import (
+	"fmt"
+	"testing"
+
+	nropv1 "github.com/openshift-kni/numaresources-operator/api/numaresourcesoperator/v1"
+
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var (
+	defaultNROSchedulerName      = "numaresourcesscheduler"
+	defaultNROSchedulerNamespace = "openshift-numaresources"
+	defaultImageSpec             = "test-registry.redhat.io/openshift4/noderesourcetopology-scheduler-rhel9:v4.14.0"
```

Comment:
Can we change this to something generic ? In unit tests it should work without issue

Reply:
it is generic, but I can change it to the dummy format